### PR TITLE
osdocs9673: creating sigstore modules

### PR DESCRIPTION
**Pull request by DCChadwick**
Version(s): 4.16+

Issue: [OSDOCS-9673](https://issues.redhat.com//browse/OSDOCS-9673)

Link to docs preview: https://76613--ocpdocs-pr.netlify.app/openshift-enterprise/latest/nodes/nodes-sigstore-using.html

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by sunilcio** [comment]
@lyman9966 could you help review? thanks

File: modules/nodes-sigstore-using-about.adoc [hunk 1]
```diff
@@ -0,0 +1,9 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-sigstore-using.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="nodes-sigstore-using-about_{context}"]
+= About the node metrics dashboard
+
+The Sigstore project enables developers to sign-off on what they build and administrators to verify signatures and monitor workflows at scale. With the SigStore project, signatures can be stored in the same registry as the build images. A second server is not needed. The identity piece of a signature is tied to the OpenID Connect (OIDC) identity through the Fulcio certificate authority, which simplifies the signature process by allowing key-less signing. Additionally, SigStore includes Rekor, which records signature metadata to an immutable, tamper-resistant ledger.
```
Comment (saschagrunert) [inline modules/nodes-sigstore-using-about.adoc:9]: I think it's just `sigstore`

```suggestion
The sigstore project enables developers to sign-off on what they build and administrators to verify signatures and monitor workflows at scale. With the sigstore project, signatures can be stored in the same registry as the build images. A second server is not needed. The identity piece of a signature is tied to the OpenID Connect (OIDC) identity through the Fulcio certificate authority, which simplifies the signature process by allowing key-less signing. Additionally, sigstore includes Rekor, which records signature metadata to an immutable, tamper-resistant ledger.
```

Comment (DCChadwick) [reply modules/nodes-sigstore-using-about.adoc:9]: Oops! I think I was trying to camel-case it for some reason. I'll fix that. Thanks!

**Review by saschagrunert — APPROVED** [review]
/lgtm

**Comment by opayne1** [comment]
Hi @DCChadwick I am relabeling your PR to send for peer review. PRs must be submitted for peer review before getting submitted for merge review.

**Comment by DCChadwick** [comment]
@opayne1 - Sorry about that! Sascha Grunert reviewed the docs and I thought he was the peer reviewer. (I didn't realize he isn't an OCP writer.) My mistake. Thank you for catching that.

File: nodes/nodes-sigstore-using.adoc [hunk 2]
```diff
@@ -0,0 +1,17 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="nodes-sigstore-using"]
+= Node metrics dashboard
```
Comment (mburke5678) [inline nodes/nodes-sigstore-using.adoc:3]:
```suggestion
= Manage secure signature with sigstore
```

**Comment by mburke5678** [comment]
@DCChadwick A few comments. Otherwise LGTM

File: nodes/nodes-sigstore-using.adoc (hunk 2 above)
Comment (DCChadwick) [reply nodes/nodes-sigstore-using.adoc:3]: Oh the joys of copy/paste. Sorry about that but thanks for catching it.

File: _topic_maps/_topic_map.yml [hunk 3]
```diff
@@ -2623,6 +2623,8 @@ Topics:
     File: nodes-sno-worker-nodes
 - Name: Node metrics dashboard
   File: nodes-dashboard-using
+- Name: Manage secure signature with SigStore
```
[the remark this answers is not in the document]
Comment (DCChadwick) [reply _topic_maps/_topic_map.yml:2626]: thanks!

File: nodes/nodes-sigstore-using.adoc [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+include::_attributes/common-attributes.adoc[]
+:context: nodes-sigstore-using
+
+toc::[]
+
+Beginning with version 4.16, {product-title} supports the sigstore project to improve supply chain security.
```
[the remark this answers is not in the document]
Comment (DCChadwick) [reply nodes/nodes-sigstore-using.adoc:9]: I like! Thank you!

File: nodes/nodes-sigstore-using.adoc [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+:context: nodes-sigstore-using
+
+toc::[]
+
+You can use the sigstore project with {product-title} to improve supply chain security.
+
+// The following include statements pull in the module files that comprise
+// the assembly. Include any combination of concept, procedure, or reference
+// modules required to cover the user story. You can also include other
+// assemblies.
```
Comment (skopacz1) [inline nodes/nodes-sigstore-using.adoc:14]: I'm not going to hold up merge review for this but just a heads up that I think you left some template content in this assembly

**Review by skopacz1 — COMMENTED** [review]
Just one comment to keep in mind for a potential future PR, but it's nothing pressing imo. Proceeding to merge

**Pull request merged**